README.md

```diff
@@ -59,7 +64,7 @@ Deployment scripts are located in `migrations/src`. To run, start `ganache` or `
 *NB: You should locally use the latest version of `ganache-cli`, as contracts rely on recent opcodes*
 
 ```
-$ ganache-cli -p 7545
+$ ganache-cli -p 7545 -l 2000000
```

Comment:
Should be `ganache-cli -p 7545 -l 8000000`, 2 million gas is too low and wouldn't allow the contracts to be deployed

Reply:
Thanks, have updated it.